boards/calliope-mini/include/periph_conf.h

```diff
@@ -36,7 +36,7 @@ extern "C" {
 #define CLOCK_HFCLK         (16U)           /* set to  0: internal RC oscillator
                                                       16: 16MHz crystal
                                                       32: 32MHz crystal */
-#define CLOCK_LFCLK         (0)             /* set to  0: internal RC oscillator
+#define CLOCK_LFCLK         (0U)             /* set to  0: internal RC oscillator
```

Comment:
I would prefer to move the changes unrelated to the actual PR (marking numbers as unsigned) into at least their own commit, if not their own PR...

Reply:
This sounds right. I'll keep it in mind when squashing, if this is ok with you.